installation/curl-autoconf-archive.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# Description: This script downloads and installs the autoconf-archive macros
+#   necessary to configure this project. This is invoked for distro builds where
+#   the necessary package is not available to install, and is also useful to
+#   hand run under certain circumstances.
+#   Should be run from the top level directory
+#
+
+# autoconf-archive url
+autoconf_archive_url="http://git.savannah.gnu.org/gitweb/?p=autoconf-archive.git;a=blob_plain;f=m4"
```

Comment:
I think this decision effectively summarises to:

- do we want to remain on the development tip of autoconf-archive (gitweb)? or,
- do we want stability at the expense of using old code (ftp)?

I'd leaning slightly towards the former since otherwise we have something else to maintain (if we go with the latter we should add the package version to `versions.txt`). If the GNU tip does break, we can raise a PR to revert to the last good commit. That said, I can't imagine that the two m4 files we're using are going to change significantly enough to warrant tracking the top of development. However, by remaining current, we'll ensure RHEL/CentOS are more in line with the other distros (which presumably will update the `autoconf-archive` package as new versions are released upstream.